index.dd

```diff
@@ -94,6 +94,25 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE Start a minimal web server,
+----
+#!/usr/bin/env dub
+/+ dub.sdl:
+name "hello_vibed"
+dependency "vibe-d" version="~>0.8.0"
```

Comment:
Vibe.d 0.8.0 will be released very [soon](http://forum.dlang.org/post/oj198q$1j9q$1@digitalmars.com), so I already accounted for the approval / review lag.